src/components/password-modal/password-modal-component.tsx

```diff
@@ -482,12 +482,12 @@ export class PasswordModalComponent extends React.Component<
         }
     }
 
-    private async verifyPassword(value: string): Promise<boolean> {
+    private async verifyPassword(value: string, biometricLogin: boolean): Promise<boolean> {
         if (Platform.OS === 'web') {
             return true;
         } else {
             try {
-                return await verifyPinInput(value);
+                return await verifyPinInput(value, biometricLogin);
```

Comment:
we should keep it more clear.
do something like this:
```
const pinCode = biometricLogin ? await getPincode() : value; // this will trigger biometric auth
return await verifyPinCode(pinCode); // verification function doesn't have to know how the pin was collected
```